documentation/src/docs/deployment/configuration-and-provisioning.md

```diff
@@ -0,0 +1,380 @@
+# Pravega Configuration and Provisioning Guide
+
+This document summarizes the most important aspects to consider for configuring and provisioning a
+production Pravega Cluster in an infrastructure-agnostic way. Note that this document does not cover all the 
+parameters in the configuration of Pravega as they are [already documented](/config.properties). Similarly,
+this is not a guide to [deploy Pravega on Kubernetes](https://github.com/pravega/pravega-operator).
+
+* [Configuring Cluster Service Dependencies](#configuring-cluster-service-dependencies)
+* [Number of Segment Containers](#number-of-segment-containers)
+* [Segment Store Cache Size and Memory Settings](#segment-store-cache-size-and-memory-settings)
+* [Durable Log Configuration: Bookkeeper](#durable-log-configuration-bookkeeper)
+* [Right-Sizing Long-Term Storage](#right-sizing-long-term-storage)
```

Comment:
This is a strange order. Perhaps we should order should reflect the order in which things will likely be tuned.
IE: starting with "Tier-2 options", then "replication factor", and "how many hosts to deploy / how to allocate resources between services" and ending with performance tuning settings.

Reply:
In the navigation panel, I tried to stick to the order you have suggested.